activerecord/test/cases/adapters/mysql2/explain_test.rb

```diff
@@ -1,23 +1,23 @@
 # frozen_string_literal: true
 
 require "cases/helper"
-require "models/developer"
-require "models/computer"
+require "models/author"
+require "models/post"
 
 class Mysql2ExplainTest < ActiveRecord::Mysql2TestCase
-  fixtures :developers
+  fixtures :authors
 
   def test_explain_for_one_query
-    explain = Developer.where(id: 1).explain
-    assert_match %(EXPLAIN for: SELECT `developers`.* FROM `developers` WHERE `developers`.`id` = 1), explain
-    assert_match %r(developers |.* const), explain
+    explain = Author.where(id: 1).explain
+    assert_match %(EXPLAIN for: SELECT `authors`.* FROM `authors` WHERE `authors`.`id` = 1), explain
```

Comment:
Why are you changing mysql2's tests only from Developer to Author?

Reply:
The original author wanted to keep the EXPLAIN with the STAR and for that, the model had to be changed.

I can update the EXPLAIN test to keep the Developer model and update the regexp to match against the new query.

What do you think?

Reply:
I think it is better to fix all other explain tests to use Author to not have a fragile test that will have to change every time the list of ignored column at Developers change.

Reply:
I'm thinking in creating a specific model for that feature, in that way the model will be exclusive for that feature and won't break anything around that.

Or even use something like that:

```ruby
test '...' do
  begin
    Developer.ignored_columns = [:first_name, :last_name]

    assertions here
  ensure
    Developer.ignored_columns = []
  end
end
```

What you think?

Reply:
I think changing to author is a good solution already that doesn't require adding any new code.

Reply:
So change the other specs to use author too?

Reply:
Yeah :+1: